opta/core/gcp.py

```diff
@@ -173,3 +173,22 @@ def bucket_exists(self, bucket_name: str) -> bool:
         except NotFound:
             return False
         return True
+
+    @classmethod
+    def get_detailed_config_map(cls) -> Dict[str, Dict[str, str]]:
+        prefix = "opta_config/"
+        credentials, project_id = cls.get_credentials()
+        storage_client = storage.Client(project=project_id, credentials=credentials)
+        opta_config_detailed_map = {}
+        for bucket in storage_client.list_buckets():
+            detailed_config = {}
+            for response in storage_client.list_blobs(
+                bucket.name, prefix=prefix, delimiter="/"
+            ):
+                blob = storage.Blob(response.name, bucket)
```

Comment:
if you compare this new code and the existing code here, you can see the opportunity of creating a private method for loading a config file based on bucket and a file name.
https://github.com/run-x/opta/blob/498aa95bacb9d52a919ed13dd40a9581cffdbd76/opta/core/gcp.py#L83-L90

let me know if you need help with addressing this comment, but this is important that we favor reusing code vs adding new whenever possible.

Reply:
Addressed this.